.eslintrc.json

```diff
@@ -17,6 +17,26 @@
   },
   "plugins": ["react"],
   "rules": {
-    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }]
-  }
+    "import/extensions": [
+      "error",
+      "ignorePackages",
+      {
+        "js": "always",
+        "jsx": "always"
+      }
+   ],
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+    "func-style": ["error", "declaration", { "allowArrowFunctions": true }]
+  },
+  "prettier/prettier": [
+    "error", 
+    {
+      "bracketSpacing": true,
+      "semi": true,
+      "singleQuote": true,
+      "trailingComma": "none",
+      "printWidth": 80,
+      "tabWidth": 2
+    }
+  ]
```

Comment:
can you double-check these changes on my end I'm getting  `Error: ESLint configuration in .eslintrc.json is invalid:
        - Unexpected top-level property "prettier/prettier".` I don't know if it would be necessary to specify prettier rules again in eslintrc.json since we already have a .prettierrc file

Reply:
Forgot to remove those from the .prettierrc file but it seemed that the prettier extension of the vscode was not reading them at all. I found it better to put them in .eslintrc.json file. Let me revise this and find a perfect solution

Reply:
about the extensions, I have seen that it is recommended to temporary disable Eslint or prettier extensions to test that formatting fixes are actually happening because of the rules in `eslintrc.json` or `.prettierrc` not being done by extensions themselves. it is to make sure the rules will work in case someone doesn't have those extensions